Samples/Widgets/MultipleWidgets/cpp-win32-packaged/SampleWidgetProviderAppPackage/Package.appxmanifest

```diff
@@ -0,0 +1,124 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  xmlns:uap3="http://schemas.microsoft.com/appx/manifest/uap/windows10/3"
+  xmlns:com="http://schemas.microsoft.com/appx/manifest/com/windows10"
+  xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
+  IgnorableNamespaces="uap uap3 rescap">
+
+	<Identity
+	  Name="SampleWidgetProviderAppPackage"
+	  Publisher="CN=Microsoft Corporation"
+	  Version="1.0.0.0" />
+
+	<Properties>
+		<DisplayName>SampleWidgetProviderAppPackage</DisplayName>
+		<PublisherDisplayName>Microsoft Corporation</PublisherDisplayName>
+		<Logo>Images\StoreLogo.png</Logo>
+	</Properties>
+
+	<Dependencies>
+		<TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.17763.0" MaxVersionTested="10.0.19041.0" />
+		<TargetDeviceFamily Name="Windows.Desktop" MinVersion="10.0.17763.0" MaxVersionTested="10.0.19041.0" />
+	</Dependencies>
+
+	<Resources>
+		<Resource Language="x-generate"/>
+	</Resources>
+
+	<Applications>
+		<Application Id="App"
+		  Executable="$targetnametoken$.exe"
+		  EntryPoint="$targetentrypoint$">
+			<uap:VisualElements
+			  DisplayName="SampleWidgetProviderAppPackage"
+			  Description="SampleWidgetProviderAppPackage"
+			  BackgroundColor="transparent"
+			  Square150x150Logo="Images\Square150x150Logo.png"
+			  Square44x44Logo="Images\Square44x44Logo.png">
```

Comment:
added the AppListEntry property

Reply:
@riverar @almedina-ms - `AppListEntry="none"` is reasonable for `Release` builds, i.e. `production` but for `Debug` builds it might be causing a headache if manual uninstall is required for testing :)
Instead of adding it here, I'd mention it as a recommendation in `README.md` :)